app/components/common/ListItem.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { View, StyleSheet, TouchableOpacity, Dimensions } from 'react-native'
+import { scale } from 'react-native-size-matters'
+import RNText from './RNText'
+import colors from '../../config/colors'
+import Icon from 'react-native-vector-icons/Ionicons'
+
+const windowWidth = Dimensions.get('window').width
+const chevronIcon = (
+  <Icon
+    name="chevron-forward-outline"
+    size={windowWidth * 0.05}
+    color={colors.white}
+  />
+)
+
+const ComponentCell = ({ title, subTitle, onPress }) => {
+  return (
```

Comment:
Usability I think we should have a check here. If onPress is null change the `activeOpacity` of the touchable so that it doesn't look perusable.